app/app.js

```diff
@@ -8,6 +9,41 @@ import config from "timed/config/environment";
 import "simplebar";
 import "simplebar/dist/simplebar.css";
 
+if (config["@sentry/ember"]) {
+  const sentryConfig = config["@sentry/ember"].sentry;
+  Sentry.init({
+    dsn: sentryConfig.dsn,
+    ignoreErrors: config.SENTRY_IGNORE,
+    tracesSampleRate: 1.0,
+    debug: sentryConfig.debug,
```

Comment:
Maybe we could add [offline error capturing as well](https://docs.sentry.io/platforms/javascript/guides/ember/configuration/transports/?original_referrer=https%3A%2F%2Fgithub.com%2Fgetsentry%2Fsentry-javascript%2Ftree%2Fmaster%2Fpackages%2Fember) 
`  transport: Sentry.makeBrowserOfflineTransport(Sentry.makeFetchTransport)`